frontend/src/components/Header.tsx

```diff
@@ -26,8 +26,10 @@ export default function Header(props: Props) {
             <Toolbar sx={{justifyContent: "center"}}>
                 <div>
                     <Typography variant="h6" component="div" align="center">
-                        Hier entsteht meine Video Game Library
+                        Die Video Game Library
                     </Typography>
+                    <br/>
+                    <br/>
```

Comment:
you could use something like: `sx={{marginBottom: 2px}}` or padding instead of <br/>

Reply:
Haha yeah was just a short edit for the Video Record for the Weekly tomorrow